[DOC-117] add feature support for YSQL Connection Manager (#19100)
yugabyte/yugabyte-db: docs/content/preview/explore/connection-manager/connection-mgr-ysql.md

To start a YugabtyeDB cluster with YSQL Connection Manager, set the [yb-tserver](../../../reference/configuration/yb-tserver/) flag `enable_ysql_conn_mgr` flag to true.

{{< note title = "Note">}}

> This feature is currently unsupported for [YugabyteDB Anywhere](../../../yugabyte-platform/) and [YugabyteDB Managed](../../../yugabyte-cloud/).

Reviewer: ```suggestion
YSQL Connection Manager is currently not supported for [YugabyteDB Anywhere](../../../yugabyte-platform/) and [YugabyteDB Managed](../../../yugabyte-cloud/).
```

> This feature is currently unsupported for [YugabyteDB Anywhere](../../../yugabyte-platform/) and [YugabyteDB Managed](../../../yugabyte-cloud/).

Reviewer: I'd move this up to the end of the opening section
